apps/secret/consolidate/v1beta1/consolidate.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package consolidatev1beta1;
+
+option go_package = "github.com/grafana/grafana/apps/secret/consolidate/v1beta1;consolidatev1beta1";
+
+import "google/protobuf/empty.proto";
+
+message SecretsConsolidateResponse {
+    bool success = 1;  // true if successful, false if error
```

Comment:
Do we want to return the error message here?

Reply:
isn't that right under this? :D

Reply:
xD